eva/utils/single_dispatch_method.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+if sys.version_info >= (3, 8):
```

Comment:
Add note explaining the reasoning behind the python version check

Reply:
done.